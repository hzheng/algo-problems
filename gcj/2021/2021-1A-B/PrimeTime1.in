100
5
2 2
3 1
5 2
7 1
11 1
2
2 2
3 1
1
17 2
1
2 7
3
2 7
17 1
241 1
8
2 1
3 1
5 1
7 2
317 1
353 1
379 1
421 1
6
2 2
5 1
7 1
19 1
23 4
29 1
8
2 2
3 1
13 1
47 1
97 1
383 1
431 2
487 1
10
2 1
17 1
163 1
167 1
353 1
409 1
439 1
443 1
467 1
491 1
8
2 1
7 1
23 1
401 1
439 1
463 3
467 1
487 1
7
3 2
43 1
149 1
263 1
347 1
367 1
487 3
8
2 1
5 2
7 1
79 1
337 1
421 1
457 1
461 1
10
2 1
3 1
5 1
13 1
17 1
59 1
383 1
439 1
457 1
461 1
6
2 1
3 2
29 1
71 1
331 2
463 3
7
3 3
5 1
229 1
313 1
401 2
439 1
499 1
9
3 1
5 1
197 1
347 1
353 1
397 1
409 1
479 2
491 1
8
2 2
3 1
7 1
17 1
31 1
67 1
491 1
499 1
7
2 3
37 1
53 1
59 1
463 2
479 1
491 1
6
7 1
107 1
337 1
397 1
449 2
479 3
6
29 1
131 1
401 2
449 1
463 3
499 1
7
2 2
3 3
137 1
257 1
443 1
467 1
487 1
7
29 1
43 2
61 1
73 1
79 1
101 1
421 1
4
2 5
3 3
431 1
433 1
4
83 1
487 1
491 4
499 3
8
2 1
3 1
101 1
179 1
229 1
251 1
283 1
331 1
5
3 4
5 3
23 1
449 1
499 1
7
3 1
5 1
23 1
179 1
307 1
359 1
499 4
8
3 1
11 1
19 1
83 1
409 1
449 2
457 2
499 1
3
2 1
3 1
5 1
4
2 2
5 4
23 1
487 2
7
2 1
71 1
311 1
373 1
431 1
433 1
467 3
6
2 1
3 1
5 4
179 1
347 1
431 1
5
2 1
3 5
59 1
431 1
467 1
5
2 1
53 1
433 5
439 1
499 1
5
2 1
3 1
31 1
47 1
59 1
4
29 1
139 1
373 1
457 1
7
2 1
5 2
41 1
271 1
379 1
439 1
499 3
10
13 1
19 1
41 1
67 1
101 1
103 1
181 1
223 1
229 1
251 1
8
3 2
29 1
311 1
397 1
421 1
467 1
487 1
499 2
3
2 8
5 1
251 1
4
5 1
79 1
367 1
439 4
7
2 1
3 1
17 2
19 1
47 1
461 3
491 1
8
2 1
7 1
31 1
421 1
479 3
487 1
491 1
499 1
7
3 1
11 2
13 1
23 2
227 1
439 1
443 2
6
2 1
3 2
11 1
127 1
311 1
491 4
6
19 1
157 1
179 1
433 1
463 3
491 2
8
7 2
41 1
97 2
239 1
283 1
383 1
449 1
461 1
5
29 1
127 1
173 1
443 3
491 3
3
2 4
3 3
433 2
6
2 1
3 1
5 2
13 3
461 1
467 1
7
3 3
29 1
241 1
397 1
409 1
433 1
449 2
7
2 2
3 2
7 2
401 1
433 1
439 1
491 1
8
3 2
349 1
373 1
383 1
457 1
463 1
487 2
491 1
6
2 1
3 2
43 2
107 1
439 1
467 3
3
3 8
463 1
487 1
5
3 2
373 1
443 2
491 3
499 2
5
3 1
17 1
67 1
487 5
491 2
6
2 2
3 3
5 2
71 1
383 1
443 1
8
3 2
17 1
109 1
113 1
229 1
409 1
463 2
487 1
7
3 1
5 1
31 1
181 1
241 1
479 1
491 4
8
3 2
53 1
271 1
277 1
311 1
401 1
479 2
499 1
7
19 1
23 1
47 1
149 1
431 1
449 1
491 3
5
2 1
239 1
467 2
479 3
491 2
8
2 1
19 1
23 1
29 1
31 1
37 1
61 1
113 1
5
2 1
3 2
11 1
433 1
439 4
7
2 2
3 3
109 1
199 1
293 1
311 1
499 1
10
2 1
3 1
5 1
7 1
11 1
13 1
17 1
19 1
23 1
29 1
7
2 1
5 1
257 1
313 1
379 1
409 1
457 2
8
2 2
3 1
67 1
193 1
373 1
443 1
467 2
499 1
6
151 1
179 1
227 1
281 1
331 1
337 1
7
2 1
3 1
31 1
331 1
457 1
491 1
499 3
6
2 4
3 2
11 1
181 1
293 1
499 1
8
2 1
3 1
5 2
41 1
373 1
443 2
449 1
479 1
3
19 1
23 2
29 6
8
7 2
23 1
41 1
67 1
97 1
367 1
457 1
499 2
5
2 1
5 1
179 1
479 1
491 5
6
5 2
41 1
59 1
239 1
433 1
499 3
4
31 2
53 1
89 1
499 5
7
2 1
17 3
269 1
281 1
307 1
479 1
499 1
5
2 4
3 3
13 1
449 1
479 1
1
499 2
7
2 2
3 2
5 1
163 1
373 1
379 2
491 1
7
2 1
3 1
37 1
389 1
421 1
439 1
479 3
8
2 1
3 1
5 2
379 2
431 1
439 1
457 1
491 1
8
2 1
5 2
37 1
293 1
419 1
421 1
443 2
463 1
9
2 1
3 1
5 1
263 1
317 1
337 1
457 2
467 1
499 1
2
7 1
499 8
5
2 5
3 1
101 1
397 1
487 1
7
3 2
331 1
347 1
349 1
389 1
397 1
499 3
9
3 1
13 1
23 1
29 1
43 1
59 1
251 1
317 1
499 2
8
3 1
5 1
43 1
67 1
167 1
463 1
479 2
487 2
7
2 3
31 1
59 1
227 1
313 1
467 2
479 1
2
443 1
491 1
10
2 1
19 1
97 1
113 1
179 1
197 1
317 1
419 1
449 1
487 1
9
2 1
3 1
5 1
7 1
233 1
313 1
467 1
491 2
499 1
7
3 2
13 2
233 1
269 1
389 1
431 1
491 2
8
31 2
47 1
211 1
367 1
431 1
439 1
487 1
499 1
7
2 1
7 1
83 1
317 2
367 1
461 1
463 3
6
2 2
11 1
71 1
181 1
397 1
491 2
9
31 1
59 1
79 1
101 1
199 1
419 1
433 1
467 1
491 1
