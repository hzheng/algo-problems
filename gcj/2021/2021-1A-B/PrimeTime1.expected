Case #1: 25
Case #2: 0
Case #3: 17
Case #4: 8
Case #5: 0
Case #6: 1470
Case #7: 140
Case #8: 1833
Case #9: 2771
Case #10: 2807
Case #11: 2367
Case #12: 0
Case #13: 1770
Case #14: 2059
Case #15: 2061
Case #16: 2955
Case #17: 0
Case #18: 1961
Case #19: 2779
Case #20: 0
Case #21: 1542
Case #22: 0
Case #23: 864
Case #24: 0
Case #25: 0
Case #26: 499
Case #27: 2685
Case #28: 2739
Case #29: 5
Case #30: 0
Case #31: 0
Case #32: 0
Case #33: 0
Case #34: 0
Case #35: 0
Case #36: 0
Case #37: 2195
Case #38: 0
Case #39: 2799
Case #40: 256
Case #41: 1835
Case #42: 1938
Case #43: 2947
Case #44: 1587
Case #45: 2286
Case #46: 2983
Case #47: 2009
Case #48: 0
Case #49: 0
Case #50: 0
Case #51: 2169
Case #52: 1764
Case #53: 3141
Case #54: 1926
Case #55: 487
Case #56: 3357
Case #57: 3417
Case #58: 900
Case #59: 2061
Case #60: 2715
Case #61: 2493
Case #62: 0
Case #63: 0
Case #64: 0
Case #65: 0
Case #66: 1308
Case #67: 0
Case #68: 1895
Case #69: 2316
Case #70: 0
Case #71: 0
Case #72: 499
Case #73: 1865
Case #74: 0
Case #75: 2009
Case #76: 0
Case #77: 0
Case #78: 0
Case #79: 0
Case #80: 479
Case #81: 499
Case #82: 1630
Case #83: 2334
Case #84: 2155
Case #85: 2105
Case #86: 2335
Case #87: 3493
Case #88: 0
Case #89: 2979
Case #90: 1677
Case #91: 2505
Case #92: 1816
Case #93: 0
Case #94: 2147
Case #95: 2191
Case #96: 2097
Case #97: 0
Case #98: 2569
Case #99: 1562
Case #100: 0
